Fix admin.conversations.restrictAccess.* methods to match documentation

ruberVulpes opened the pull request:
## Summary

I wanted to make these changes to get auto-fill when using the slackclient in my IDE. I based the required/optional parameter style off of `chat_postMessage`

It also looks like when I ran `python setup.py validate` black formatted a few more files than I would expect

* `admin_conversations_restrictAccess_addGroup`
  * Added arguments `channel_id` and `group_id`
  * Added doc strings for arguments based on [documentation ](https://api.slack.com/methods/admin.conversations.restrictAccess.addGroup)

* `admin_conversations_restrictAccess_listGroups`
  * Added arguments `channel_id` 
  * Added doc strings for arguments based on [documentation ](https://api.slack.com/methods/admin.conversations.restrictAccess.listGroups)

* `admin_conversations_restrictAccess_removeGroup`
  * Added arguments `channel_id`, `group_id`, and `team_id`
  * Added doc strings for arguments based on [documentation ](https://api.slack.com/methods/admin.conversations.restrictAccess.removeGroup)

### Category (place an `x` in each of the `[ ]`)

- [x] **slack.web.WebClient** (Web API client)
- [ ] **slack.webhook.WebHookClient** (Incoming Webhook, response_url sender)
- [ ] **slack.web.classes** (UI component builders)
- [ ] **slack.rtm.RTMClient** (RTM client)

## Requirements (place an `x` in each `[ ]`)

- [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/python-slackclient/blob/main/.github/contributing.md) and have done my best effort to follow them.
- [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
- [x] I've run `python setup.py validate` after making the changes.

seratch commented:
Thanks a lot. Indeed, the list of required args in docs is not different. Could you check the following?

* I don't know why your black's modifications are different from my one but please revert the formatted changes to unrelated source files (they're causing some CI failures)
* Update `tests/web/test_web_client_coverage.py` accordingly

seratch reviewed (changes requested):
As I already mentioned in the previous comment, this PR still needs some updates

ruberVulpes commented:
* Reverted the formatting changes to unrelated source files
  * Not sure why I didn't think of this when I pushed the changes
* Updated `tests/web/test_web_client_coverage.py` based on other entries

CI builds look good now 👍

seratch commented:
@ruberVulpes Thanks - looks great to me.
